'''
Напишите функцию division(a, b), которая принимает на вход два числа, делимое (a) и делитель (b), и возвращает частное.
Необходимо реализовать внутри функции отлов исключения ZeroDivisionError на случай, если пользователь при вызове функции попытается поделить на ноль.
Если в процессе выполнения функции было поймано исключение ZeroDivisionError, то функция должна вывести на экран сообщение "Zero division error!" с помощью функции print(), а затем вернуть значение None из функции.
Примеры работы программы:
'''

a = 10
b = 0
def division(a, b):
    try:
        return a / b
    except ZeroDivisionError:
        print("Zero division error!")
        return None






